addons/mail/static/src/models/file_uploader_view/file_uploader_view.js

```diff
@@ -175,7 +175,6 @@ registerModel({
         thread: many2one('Thread', {
             compute: '_computeThread',
             readonly: true,
-            required: true,
```

Comment:
why this change?

Reply:
I had failure in tests that `_computeThread` made `replace(undefined)` with `Chatter` as owner.
`Chatter/thread` is not necessarily set on `chatter`, so I removed the `required` because it's wrong in this case.

Reply:
What kind of chatter doesn't have a thread suddenly with this PR? This is a mistake from the test then, we shouldn't need to change the code.

Reply:
Ok, will do the change to enforce chatter having a thread. I suppose we didn't set it `required` and have conditional on `Chatter/thread` due to how we dealt with temporary threads or waiting for `init_messaging` RPC on page load to have a thread... But now we know JS modelling is designed with incomplete knowledge so it makes sense to have `Chatter/thread` relation immediately even though info on `thread` could be lacking.

Reply:
If we model "temporary thread" chatter differently (eg. by having empty thread), then yes it could become not required, but to do that correctly imply a lot of changes in the whole code of chatter and related components and models, so way out of scope here.

Reply:
Hm... looks like I didn't have to do anything to make test pass. I see that insert of `Chatter/thread` come from onChange `_onThreadIdOrThreadModelChanged` so possibly had the issue due to nested update cycle that I just removed...

I won't risk killing this onChange in this PR with all current changes. It works at the moment so it's safe to not touch it for the time being.